acceptance/localcluster/localcluster.go

```diff
@@ -324,13 +324,17 @@ func (l *Cluster) processEvent(e dockerclient.EventOrError) bool {
 	defer l.mu.Unlock()
 
 	if e.Error != nil {
-		l.Events <- Event{NodeIndex: -1, Status: EventDie}
+		if l.Events != nil {
+			l.Events <- Event{NodeIndex: -1, Status: EventDie}
```

Comment:
Is the error itself logged elsewhere? Seems like it would be pretty useful to surface it if not.